rs/ethereum/cketh/mainnet/README.md

```diff
@@ -200,7 +200,7 @@ pub fn setup(env: TestEnv) {
         env.topology_snapshot(),
         NnsCustomizations::default(),
     );
-    env.sync_with_prometheus();
+    env.sync_with_prometheus("");
```

Comment:
It's not immediately clear what a boundary node name of `""` means. What's the semantic?

Reply:
I changed signature to `Option<&str>` and added doc comment. Hope that makes it better.